recipes/coin-cgl/all/conanfile.py

```diff
@@ -0,0 +1,121 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+from contextlib import contextmanager
+import os
+
+
+class CoinCglConan(ConanFile):
+    name = "coin-cgl"
+    description = "COIN-OR Cut Generator Library"
+    topics = ("conan", "clp", "simplex", "solver", "linear", "programming")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/coin-or/Cgl"
+    license = ("EPL-2.0",)
+    settings = "os", "arch", "build_type", "compiler"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    exports_sources = "patches/**.patch"
+    generators = "pkg_config"
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.settings.os == "Windows" and self.options.shared:
+            raise ConanInvalidConfiguration("coin-cgl does not support shared builds on Windows")
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("coin-utils/2.11.4")
+        self.requires("coin-osi/0.108.6")
+        self.requires("coin-clp/1.17.6")
+
+    def build_requirements(self):
+        self.build_requires("pkgconf/1.7.3")
```

Comment:
pkgconf comes for free in msys2 package (as well as autoconf, automake etc), but I know that you prefer to have full control over which pkgconf binary is used inside the recipe.

Reply:
As long as `base-devel` is installed: https://packages.msys2.org/package/pkg-config
Which the default msys2 does:
https://github.com/conan-io/conan-center-index/blob/88a19d55ad885c9b7f14918bd428cd424f9d18f3/recipes/msys2/all/conanfile.py#L20

Better be safe then sorrow, like meson and ninja